### DISCUSSION TITLE
Task02 - Q5-e

### DISCUSSION THREAD

**Discussion by SMRazaRizvi96** [chulminy/CIVE497-CIVE700#153] [Q&A] [answered]
Regarding Q5-e that says:

_**(e)** Write a code to find numerical Fourier coefficients in the general and complex forms and compare them with the analytic Fourier coefficients found in (b) and (c)._  

Here, we don't need to plot the general and complex forms but only need to find the coefficients and compare them with the Analytical solutions found in b and c. Right?

**Comment by chulminy** [comment, marked as answer]
You're correct. No. You don't need to draw the graph.
